package.json

```diff
@@ -45,6 +45,7 @@
     "standard": "10.0.2",
     "uglify-js": "github:mishoo/UglifyJS2#harmony"
   },
+  "yarnDependency": "0.23.3",
```

Comment:
@dickeyxxx fixed in https://github.com/heroku/cli-engine/pull/94/commits/f0a2986703483b60291837c064eeb371df784df9